### ISSUE TITLE
Crashes with StartsOnPage 

### ISSUE THREAD

**Issue by rasatpc** [fvwmorg/fvwm3#1100]
* Fvwm3 version (run: `fvwm3 --version`)
version 1.1.1 (1.1.0-112-gf789db30)

## Expected Behaviour
To move apps at the start to an assigned page.
Style	*Firefox 	StartsOnPage	0 1 0
Style	Telegram 	StartsOnPage	1 0 0
Style	Evolution 	StartsOnPage	1 1 0

## Behaviour
StartsOnPage	0 0 0 is ok but on any other page, it crashes and logs out of system.

**Comment by ThomasAdam** [comment]
Hi @rasatpc 

It doesn't crash for me. 

Is there a corefile created? Anything in .fvwm/fvwm3-output.log ?

**Comment by somiaj** [comment]
Edit, the above option needs to be `BugOpts ExplainWindowPlacement True`, but it appears the crash is before it outputs anything (I was able to reproduce it).

**Comment by rasatpc** [comment]
I did a test with default config and added this at the bottom. All app crashes except Evolution, Gimp, and Telegram. The crash is immediate and doesn't get recorded.
```
Style	Chromium	StartsOnPage	0 1 0
Style	*Chrome 	StartsOnPage	0 1 0
Style	*Firefox 	StartsOnPage	0 1 0
Style	Evolution	StartsOnPage	1 0 0
Style	"Mozilla Thunderbird" StartsOnPage	1 1 0
Style	Gimp		StartsOnPage	2 0 0
Style	scribus		StartsOnPage	2 1 0
Style	GitHub*		StartsOnPage	2 1 0
Style	Telegram 	StartsOnPage	3 0 0
```

**Comment by somiaj** [comment]
I have tracked down where the crash is happening, but unsure of a fix. For a temporary workaround use `Style * SkipMapping`. This will work around the crash, but you will have to manually move to the page the window is started on after it starts if you want to access it.

**Comment by somiaj** [comment]
@rasatpc I think I tracked down the issue (at least for my tests). Check if the issue is fixed for you in the PR #1101.

**Comment by rasatpc** [comment]
Thanks, PR #1101 works fine.

**Comment by rasatpc** [comment]
version 1.1.1 (1.1.0-118-g5c9b0ae2), works fine without crashing.

**Comment by ThomasAdam** [comment]
@rasatpc 

Please do not close issues; there's automation in place to do this for me.

**Comment by ThomasAdam** [comment]
Fixes #1100
